src/app/containers/Bulletin/__snapshots__/index.test.jsx.snap

```diff
@@ -1304,7 +1303,7 @@ exports[`Bulletin Container snapshots should render a TV bulletin correctly 1`]
         >
           <span
             class="c6"
-            lang="en-GB"
+            lang="ig"
```

Comment:
Is this right? The language attribute is set on the document, we only then need to apply this againt if the content has a different language to the docuement

Reply:
https://github.com/bbc/simorgh/pull/6330#discussion_r417211300